requests/Coupon.php

```diff
@@ -29,6 +29,7 @@ public function rules()
             ['order_restriction', 'igniter.coupons::default.label_order_restriction', 'integer'],
             ['status', 'admin::lang.label_status', 'boolean'],
             ['locations.*', 'admin::lang.column_location', 'integer'],
+            ['locations', 'admin::lang.column_location', 'required'],
```

Comment:
For the sake of consistency, the `locations` rule should come before `locations.*`